src/rust/bitbox02/src/backup.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 Shift Crypto AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+extern crate alloc;
+use alloc::string::String;
+pub use bitbox02_sys::backup_error_t as Error;
+
+pub struct CheckData {
+    pub id: String,
+    pub name: String,
+    pub birthdate: u32,
```

Comment:
please add a `// unix timestamp` comment for people like me :)

Reply:
also, UCT or local? my guess the former since there's no offset here but it's just my guess.

Reply:
yeah, UTC. added comment